docs/analyzer/user_guide.md

```diff
@@ -292,6 +292,13 @@ output arguments:
   --print-steps         Print the steps the analyzers took in finding the
                         reported defect.
 
+environment variables:
+  CLANGSA_PLUGIN_DIR       If the CC_ANALYZERS_FROM_PATH environment variable
```

Comment:
Shouldn't we use `CC_` prefix for this environment variable too?

Reply:
Yes, nice catch!